.circleci/config.yml

```diff
@@ -30,6 +30,20 @@ jobs:
       command: nix-shell --run hc-rust-test
       no_output_timeout: 20m
 
+ cov-kcov:
+  machine: true
+  resource_class: xlarge
+  steps:
+   - checkout
+   - run: docker run --rm --security-opt seccomp=unconfined -v $(pwd):/project holochain/holochain-rust:circle.build.develop sh -c "cd /project && nix-shell --run hc-rust-coverage-kcov"
```

Comment:
both tarpaulin and kcov need this docker `--security-opt seccomp=unconfined` - I don't know another way of doing this with circleci - but this makes downloading the docker image slower.